tests/test_util.py

```diff
@@ -26,3 +26,19 @@ def test_split_and_preprocess(read_data):
     assert len(y_train) <= TRAIN_SIZE * len(iris_data.data)
     assert (len(X_train) == len(y_train))
     assert (len(X_test) == len(y_test))
+
+
+def test_hyperopt2skopt_space():
+    s1 = {'a': hp.uniform('a', 0, 10)}
+    s1 = modelgym.util.hyperopt2skopt_space(s1)
+    s2 = Real(0, 10)
+    s3 = {'max_depth': hp.quniform('max_depth', 2, 10, 1)}
+    s3 = modelgym.util.hyperopt2skopt_space(s3)
+    s4 = Integer(2, 10)
+    s5 = {'min_child_weight': hp.loguniform('min_child_weight', -16, 5)}
+    s5 = modelgym.util.hyperopt2skopt_space(s5)
+    s6 = Real(np.exp(-16), np.exp(5))
+
+    assert s1.get('a') == s2
+    assert s3.get('max_depth') == s4
+    assert s5.get('min_child_weight') == s6
```

Comment:
there should also be negative tests (i.e. hyperopt2skopt_space({'blah': 'blah'}) or hyperopt2skopt_space({'blah': hp.pchoice()) produce something reportedly invalid.